desc/timedesc.mli

```diff
@@ -577,6 +577,103 @@ type timestamp = Span.t
 
 (** {1 Date time components} *)
 
+(** {2 Partial date}
+
+    Implementation of:
+    - Gregorian year month ({!Ym})
+    - ISO week ({!ISO_week})
+*)
+
+module Ym : sig
+  type t
+
+  type error =
+    [ `Does_not_exist
+    | `Invalid_year of int
+    | `Invalid_month of int
+    ]
+
+  exception Error_exn of error
+
+  (** {1 Constructors} *)
+
+  val make : year:int -> month:int -> (t, error) result
+
+  val make_exn : year:int -> month:int -> t
+
+  (** {1 Accessors} *)
+
+  val year : t -> int
+
+  val month : t -> int
+
+  val year_month : t -> int * int
+
+  (** {1 Arithmetic} *)
+
+  val add : ?year:int -> ?month:int -> t -> t
+
+  val sub : ?year:int -> ?month:int -> t -> t
+
+  val diff_month : t -> t -> int
```

Comment:
Shouldn't `year` and `month` be plural here as well? Since these are counts, not specific years and months.